nmstatectl/nmstatectl.py

```diff
@@ -61,7 +61,7 @@ def setup_subcommand_set(subparsers):
                             'stdin is used when no file is specified.',
                             nargs='?')
     parser_set.add_argument(
-        '--verify', action='store_true',
+        '--no-verify', action='store_false', dest='verify', default=True,
         help='Verify that the desired state is now part of the current state.'
```

Comment:
The help text seems does not explain `no-verify`. Maybe something like: `Disable the verify of desired state is now part of the current state.`

Reply:
I would suggest: `Do not verify that the desired state is part of the current state post execution.`

Reply:
Good catch. I changed it to:

`Do not verify that the state was completely set and disable rollback to previous state.`

How do you like it?